<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSettingsWidget</class>
 <widget class="QWidget" name="SystemSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter your button assignment. 
Select your 'Enter' button.</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="circle">
         <property name="text">
          <string>Circle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="cross">
         <property name="text">
          <string>Cross</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="pstv_mode">
         <property name="text">
          <string>PS TV Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="show_mode">
         <property name="text">
          <string>Show Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="demo_mode">
         <property name="text">
          <string>Demo Mode</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
